strapi/strapi
# [DEITS] Clean up import,export,transfer commands and tests

### What does it do?

- Improves exiting and error handling for import, export, and transfer commands
  - if there is an error in the data reporting, a transfer/import/export is now still reported as successful
  - import now uses console.log instead of console.info to be consistent with the other commands
- improves tests for import and export
- adds tests for transfer that actually work
- removes the useless `logger = console` at the top of each command; if we actually switch to using a logger, we can add it back in later but for now it was only complicating things

### Why is it needed?

To improve testing, refactoring, and fix minor messaging issues for import, export, and transfer

### How to test it?

Tests should still pass
Import, export, and transfer commands should still work

Now if there is an error in the printing of the results table, the transfer is still
